.circleci/config.yml

```diff
@@ -0,0 +1,42 @@
+version: 2.1
+
+orbs:
+  node: circleci/node@5.0
+
+jobs:
+  execute-npm:
+    parameters:
+      npm-run-command:
+        type: string
+    docker:
+      - image: cimg/base:stable
+    environment:
+      SCRIVITO_TENANT: "1234567890abcdef1234567890abcdef"
+    steps:
+      - checkout
+      - node/install
+      - run:
+          name: Install correct npm version
+          command: |
+            echo "Node version"
+            node --version
+            echo ""
+            dpkg --compare-versions `npm -v` ge 7.10 || npm i -g npm@^7.10
+            echo ""
+            echo "NPM version"
+            npm --version
```

Comment:
* [ ] In the [docs](https://circleci.com/developer/orbs/orb/circleci/node) this looks simpler. Is there a reason we can't do it like that?

```yaml
    steps:
      - checkout
      - node/install:
          node-version: '16.13'
      - run: node --version
```

Reply:
I did not want to explicitly state the node version. It should be picket up by `.nvmrc`.

Once we're using node 14 as the baseline (which will be soon - support for node 12 ends in April) we can hopefully remove the "check for npm version 7.10" check.

Reply:
[nodejs 14 comes with npm@6 by default](https://github.com/nodejs/node/discussions/37689#discussioncomment-470971) it means you have to keep "check for npm version 7.10" if you want to use it